mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -1943,6 +1943,11 @@ def LLVM_CallIntrinsicOp
     attr-dict
   }];
 
+  let builders = [
+    OpBuilder<(ins "Type": $results, "StringAttr":$intrin, "ValueRange":$args)>,
```

Comment:
```suggestion
    OpBuilder<(ins "Type": $result, "StringAttr":$intrin, "ValueRange":$args)>,
```
nit: maybe call it result or resultType since it is just one type.